app/controllers/hyrax/admin/collection_types_controller.rb

```diff
@@ -44,7 +44,8 @@ def edit
 
     def update
       if @collection_type.update(collection_type_params)
-        redirect_to hyrax.edit_admin_collection_type_path(@collection_type), notice: t(:'hyrax.admin.collection_types.update.notification', name: @collection_type.title)
+        referer = hyrax.edit_admin_collection_type_path(@collection_type) + (params['referer_anchor'] || '')
```

Comment:
I realize this is a simpler case than above, but I might like to see this handled in a new private method, `update_referer`. Also prefer to use a symbol for the hash key if possible.